lib/experimental/new_embed.dart

```diff
@@ -218,7 +218,7 @@ class NewEmbed {
     morePopover = DElement(querySelector('#more-popover'));
     menuButton = DisableableButton(querySelector('#menu-button'), () {
       menu.open = !menu.open;
-    });
+    })..hidden = gistId.isEmpty;
```

Comment:
Would it be better to leave it blank to start whether there's a gistId or not?

If there's no gistId, hidden is definitely better. If there is a gistId but the gist has no test, the button will appear for a moment, but then disappear, which I (and this is personal opinion talking, rather than hard UXR skill) find more disconcerting than the opposite (hidden menu button appears when the gist is loaded and it has a test).

If I'm misunderstanding the change at 582 or you just really prefer the way you've got it, don't be shy about pushing back.

Reply:
Good point, fixed.